packages/react-hooks/src/use-map-interaction/use-map-interaction.ts

```diff
@@ -236,15 +236,20 @@ export const useMapHover = (
 
   const onMapHover = useCallback(
     (event) => {
-      // Turn all sources with active feature states off
-      cleanFeatureState()
       const hoverEvent: InteractionEvent = {
         type: 'hover',
         point: event.point,
         longitude: event.lngLat[0],
         latitude: event.lngLat[1],
       }
-      if (event.features?.length) {
+      const isLinkHover = event.target.tagName.toLowerCase() === 'a'
```

Comment:
```suggestion
      const isNotCanvas = event.target.tagName.toLowerCase() !== 'div'
```
Could this be more future-proof ie maybe at some poitn we add `<button>`s ?

Reply:
Good catch but I'd prefer at this stage (to production in 3 days) to cover only the use cases known